tests/test_ephemerisQuery.py

```diff
@@ -0,0 +1,73 @@
+# This file is part of ap_association.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+from mock import patch
+import numpy as np
+import os
+import pandas as pd
+import unittest
+
+import lsst.ap.association.ephemerisQuery as ephQ
+import lsst.geom as geom
+import lsst.pipe.base as pipeBase
+from lsst.utils import getPackageDir
+import lsst.utils.tests
+
+
+class TestEphemerisQuery(unittest.TestCase):
```

Comment:
There are a few different `requests` mock/test libraries such as `requests-mock`; I suggest using one of those to write some tests that include failure conditions (403/404 errors, data dropouts). I suspect #dm-square will have suggestions about what libraries to use, and how to implement such tests.

Reply:
I'll address it as above with the other change to requests ticket.